src/Scoringline/SendinblueApi/Model/Email.php

```diff
@@ -0,0 +1,311 @@
+<?php
+
+namespace Scoringline\SendinblueApi\Model;
+
+use Symfony\Component\HttpFoundation\File\File;
+
+class Email
+{
+    /**
+     * @var array
+     */
+    private $to = [];
+
+    /**
+     * @var array
+     */
+    private $from = [];
+
+    /**
+     * @var string
+     */
+    private $subject = '';
+
+    /**
+     * @var string
+     */
+    private $text = '';
+
+    /**
+     * @var string
+     */
+    private $html = '';
+
+    /**
+     * @var array
+     */
+    private $headers = ["Content-Type" => "text/html; charset=utf-8"];
```

Comment:
Use it in the constructor, so you can take a charset as parameter of your contruct method